src/apply-networking-linux.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
quite a useless line inmho, as some of the OSses that this scripts supports do not have "python" binary in path - see RHEL / CentOS 8 or Ubuntu 18.04. If you just want to +x this script, and then run /bin/bash script.py args, it won t work on the OSses from above.